test/lru_cache_test.rb

```diff
@@ -0,0 +1,63 @@
+require 'test_helper'
+
+class TestLRUCache < Minitest::Test
+  def setup
+    @lru_cache = LRUCache.new(5)
+  end
+
+  def test_set_item
```

Comment:
You now have two tests that do the same thing so maybe one test called `test_set_get_item`?